### ISSUE TITLE
Bug? completion fails in a very specific case

### ISSUE THREAD

**Issue by vneiger** [lervag/vimtex#1599]
Auto-completion seems to fail in the following situation (minimal working example below):
- document class: using the latest version of acmart.cls compiled from https://github.com/borisveytsman/acmart/ or the version that ships with texlive on ubuntu 18.04 (no failure with the usual "article" class)
- trying to complete a reference ``\ref{...``    (same problem with "\cref" for example)
- inserting a figure with an ordinary label, e.g. ``\label{fig:deja}``
- but, and this is the important part, with a specific caption: it must be in small caps and end with an accented letter, e.g. ``\caption{\textsc{D\'ej\`a}}`` (it does not fail with ``\caption{\textsc{D\'eja}}`` or with ``\caption{\textsc{D\'ej\`a vu}}`` or with ``\caption{D\'ej\`a}``).

In short, trying to auto-complete the ref in the following MWE:
```tex
\documentclass{acmart}
\begin{document}
%\ref{   % <== uncomment here and try to auto-complete
\begin{figure}
  \caption{\textsc{D\'ej\`a}}
  \label{fig:deja}
\end{figure}
\end{document}
```
I get an error in the bottom status pane which reads
```
Error detected while processing function vimtex#complete#omnifunc[30]..142[1]..143[1]..144[18]..145:
line   26:
E684: list index out of range: 0
```
Thanks for help!

**Comment by lervag** [comment]
I think this is fixed now. Thanks for reporting!

**Comment by vneiger** [comment]
Indeed, no failure anymore on the files I was working on. Thanks for the fast fix!

**Comment by lervag** [comment]
Great, happy to hear it!
